controllers/dashboard_controller.go

```diff
@@ -201,13 +201,15 @@ func (r *GrafanaDashboardReconciler) Reconcile(ctx context.Context, req ctrl.Req
 			continue
 		}
 
-		// first reconcile the plugins
-		// append the requested dashboards to a configmap from where the
-		// grafana reconciler will pick them up
-		err = ReconcilePlugins(ctx, r.Client, r.Scheme, &grafana, dashboard.Spec.Plugins, fmt.Sprintf("%v-dashboard", dashboard.Name))
-		if err != nil {
-			controllerLog.Error(err, "error reconciling plugins", "dashboard", dashboard.Name, "grafana", grafana.Name)
-			success = false
+		if grafana.Spec.External == nil {
```

Comment:
How do you feel about introducing a struct method for that instead?

```go
func (in *Grafana) IsInternal() bool {
	return in.Spec.External == nil
}
```

The same for external instances:
```go
func (in *Grafana) IsExternal() bool {
	return in.Spec.External != nil
}
```
or even through negation (Python protocols style):
```go
func (in *Grafana) IsExternal() bool {
	return !in.IsInternal()
}
```

Not like those are necessary, I just think it'll be easier to read the code. :)

Reply:
Agreed, I have updated to PR to match